app-backend/app/src/main/scala/scene/Routes.scala

```diff
@@ -39,9 +40,8 @@ trait SceneRoutes extends Authentication
 
   def listScenes: Route = authenticate { user =>
     (withPagination & sceneQueryParameters) { (page, sceneParams) =>
-      complete {
-        Scenes.listScenes(page, sceneParams, user)
-      }
+      RfPagination.validatePage(page)
```

Comment:
Yeah, I attempted to extend that trait and add a modified version of the `withPagination` directive with two additional cases that utilize guards to check that its not less than zero, but I think a lot of the private methods and classes were causing issues.

Pairing tomorrow would be much appreciated!